features/init.feature

```diff
@@ -8,6 +8,11 @@ Feature: hub init
     When I successfully run `hub init -g`
     Then the url for "origin" should be "git@github.com:mislav/dotfiles.git"
 
+  Scenario: Initializes a git repo in a new directory with remote
+    When I successfully run `hub init -g new_dir`
+    And I cd to "new_dir"
+    Then the url for "origin" should be "git@github.com:mislav/new_dir.git"
```

Comment:
This step would pass even if the remote configuration erroneously happened in the starting directory before you cd'd into `new_dir`. So before changing the directory, maybe we should assert that no `.git` directory was created in the current location as the result of `hub init` operation (exactly the bug that we're fixing).